internal/pkg/template/templates/environment/cf.yml

```diff
@@ -557,6 +557,62 @@ Resources:
 {{include "lambdas" . | indent 2}}
 {{include "custom-resources" . | indent 2}}
 {{- end}}
+{{- if .VPCConfig.Flowlogs }}
+  Flowlog:
+    Metadata:
+      'aws:copilot:description': 'A Flowlog for the VPC to capture information about the IP traffic'
+    Type: AWS::EC2::FlowLog
+    Properties:
+      DeliverLogsPermissionArn: !GetAtt Flowlogrole.Arn
+      LogDestinationType: cloud-watch-logs
+      LogGroupName: !Join ['-', [!Ref AppName, !Ref EnvironmentName, FlowLogGroup]]
+      MaxAggregationInterval: 60
+{{- if .VPCConfig.Imported}}
+      ResourceId: {{.VPCConfig.Imported.ID}}
+{{- else}}
+      ResourceId: !Ref VPC
+{{- end}}                 
+      ResourceType: VPC
+      TrafficType:  ALL
+  Flowlogrole:
+    Metadata:
+      'aws:copilot:description': 'A IAM Role {{- if .PermissionsBoundary}} with permissions boundary {{.PermissionsBoundary}} {{- end}} for AWS EC2 to publish flow logs to a CloudWatch log group'
+    Type: AWS::IAM::Role
+    Properties:
+      {{- if .PermissionsBoundary}}
+      PermissionsBoundary: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/{{.PermissionsBoundary}}'
+      {{- end}}
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: vpc-flow-logs.amazonaws.com
+            Action: sts:AssumeRole
+            Condition:
+              StringEquals:
+                aws:SourceAccount: !Ref AWS::AccountId
+              ArnLike:
+                aws:SourceArn:
+                  - Fn::Sub:
+                      - 'arn:aws:ec2:${region}:${account}:vpc-flow-log/*'
+                      - region: !Ref AWS::Region
+                        account: !Ref AWS::AccountId
+                        flowlogid: !Join ['-', [!Ref AppName, !Ref EnvironmentName, FlowLogGroup]]
```

Comment:
I don't see `${flowlogid}` in `'arn:aws:ec2:${region}:${account}:vpc-flow-log/*'`. Was it meant to be
```
- 'arn:aws:ec2:${region}:${account}:vpc-flow-log/*'
                      - region: !Ref AWS::Region
                        account: !Ref AWS::AccountId
```

Reply:
My bad, It was supposed to be flowlogid